### DISCUSSION TITLE
New user, probably a stupid question: has my terminal to stay open for monitoring?

### DISCUSSION THREAD

**Discussion by Taxicletter** [abraunegg/onedrive#2836] [Q&A] [answered]
I installed onedrive with brew (on Bluefin Linux). I successfully connected my onedrive. If I do `onedrive --monitor` sync is successful. But the cursor stays blinking. Do I have to leave my terminal open permanently to have my OneDrive monitored?

**Comment by abraunegg** [comment, marked as answer]
Please read all the documentation

**Reply by Taxicletter** [reply]
Can you point me in the right direction?
Here: https://github.com/abraunegg/onedrive/blob/master/docs/usage.md#ongoing-synchronisation-operational-mode-monitor-mode
I cannot find the answer.

These commands don't work on Bluefin:
`su - root`
`systemctl --user enable onedrive`
`systemctl enable onedrive`

Also, a lot of what's explained there, is above my head. Then the answer is: that's to complex for an everyday user, this is only ment for programmers.

**Reply by abraunegg** [reply]
To normally run a systemd service, please read: https://github.com/abraunegg/onedrive/blob/master/docs/usage.md#how-to-initiate-the-client-as-a-background-service

> These commands don't work on Bluefin:

Firstly - dont su to root ..... you need to enable the service for your normal user you are logging in as ...

My suggestion is you take the above URL and ask the folk who provide Bluefin support how to get this working on your platform.
